flatpak-guide.md

```diff
@@ -268,6 +268,10 @@ You already know the basics for:
     $ appstream-util validate /var/lib/flatpak/app/org.sugarlabs.ImplodeActivity/current/active/files/share/appdata/org.sugarlabs.ImplodeActivity.appdata.xml
     ```
 
+    If appstream-util validate is not running, you can also try `flatpak run org.freedesktop.appstream-glib validate` as documented in https://github.com/flathub/flathub/wiki/AppData-Guidelines#use-flathubs-appstream-util
```

Comment:
Better to link the document than showing the whole URL. e.g. "as described in the [flathub documentation](https://github.com/flathub/flathub/wiki/AppData-Guidelines#use-flathubs-appstream-util)."